internal/resolution/entitysources/catalogdsource.go

```diff
@@ -86,6 +87,8 @@ func getEntities(ctx context.Context, client client.Client) (input.EntityList, e
 				// this is already a json marshalled object, so it doesn't need to be marshalled
 				// like the other ones
 				props[property.TypePackage] = string(prop.Value)
+			case entity.PropertyBundleMediaType:
+				props[entity.PropertyBundleMediaType] = string(prop.Value)
```

Comment:
Unrelated, but we need to capture a follow-up here to make sure our FBC -> entity translation is not lossy with respect to property type. We should be forwarding all properties through.

I think we need to go back to figure out what deppy expects here. Properties can be duplicated (some are expected to be, some are not). In this case, we expect probably 0 or 1 mediatype properties. We expect exactly one package property. But then we allow 0-n for `olm.gvk`, `olm.gvk.required`, `olm.package.required`, etc.